scripts/verify-phase-18-wazuh-lab-topology.sh

```
@@ -127,4 +170,56 @@ for artifact in "${required_artifacts[@]}"; do
+  'AEGISOPS_WAZUH_AEGISOPS_SHARED_SECRET_FILE=./secrets/aegisops-wazuh-shared-secret.txt'
+  'AEGISOPS_WAZUH_GITHUB_AUDIT_ENROLLMENT_STATUS=reviewed-first-live-family-only'
+  '# Do not commit live secrets.'
+)
+
+for line in "${asset_bootstrap_required_lines[@]}"; do
+  require_fixed_string "${asset_bootstrap}" "${line}"
+done
+
+asset_integration_required_lines=(
+  '  <name>aegisops-github-audit</name>'
+  '  <hook_url>${AEGISOPS_WAZUH_AEGISOPS_INGEST_URL}</hook_url>'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)